### PR TITLE
Keycard swarm

### PR DESCRIPTION
**Pull request by guylouis**
Keycard swarm 
Focusing on software work done for Keycard and not the rest of the project (e.g production, marketing)

### CODE REVIEW

File: keycard swarm.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+id: #000-template
```
Comment (oskarth) [inline keycard swarm.md]: id and status missing

File: keycard swarm.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+title: Swarm Proposal
+status: research/specification/implementation/maintenance
+lead-contributor
```
Comment (oskarth) [inline keycard swarm.md]: same line, is YAML

same with contributors format
- foo
- bar

File: TEMPLATE.md [hunk 3]
```diff
@@ -2,10 +2,14 @@
 id: #000-template
 title: Swarm Proposal
 status: research/specification/implementation/maintenance
-lead-contributor: 
+lead-contributor: @guylouis
 contributors:
-    -
-    -
+    - Michele (@bitgamma)
+    - Andrea (@pilu)
```
Comment (gravityblast) [inline TEMPLATE.md]: @guylouis I updated my username to be @gravityblast

File: keycard swarm.md (hunk 2 above)
Comment (guylouis) [reply keycard swarm.md]: corrected

File: keycard swarm.md (hunk 1 above)
Comment (guylouis) [reply keycard swarm.md]: corrected

**Comment by oskarth** [comment]
Fixed some formatting and restored template.

**Pull request merged**